code-for-canada/project-thundercat
# [Candidate Flow] Adding user's first and last name + Updating description in dashboard page

# Description

This PR is introducing a new welcome message when the user's logged in. The dashboard is now displaying the user's first and last name and a new description.

I also improved the url restrictions when logged in. That means when you use an invalid url when logged it, it'll redirect you to the dashboard page.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Example with Registration Flow:**

![image](https://user-images.githubusercontent.com/23021242/59508841-4d45ff80-8e7d-11e9-8af8-63f66e208a15.png)

![image](https://user-images.githubusercontent.com/23021242/59508853-5767fe00-8e7d-11e9-9e73-99797748f0f7.png)

**When Logging in:**

![image](https://user-images.githubusercontent.com/23021242/59508893-74043600-8e7d-11e9-8ddf-66fc8c30173d.png)

![image](https://user-images.githubusercontent.com/23021242/59508910-7e263480-8e7d-11e9-8c99-eee16f614217.png)

**Invalid URL Example:**

![image](https://user-images.githubusercontent.com/23021242/59508969-b0d02d00-8e7d-11e9-8c68-6a73bf6fbe43.png)

_Hit enter_

![image](https://user-images.githubusercontent.com/23021242/59508979-b9c0fe80-8e7d-11e9-95bc-0795227d1e81.png)


# Testing

Manual steps to reproduce this functionality:

1.  Login and/or create a new account.
2.  Once logged in, make sure that you see your name in the welcome page.
3.  Enter invalid urls and login url and make sure that you are redirected to the dashboard page (you need to be authenticated for that).

# Checklist

Applicable for all code changes.
